### ISSUE TITLE
MacOSX

### ISSUE THREAD

**Issue by tiptronic** [HomeControlAS/cordova-plugin-advanced-websocket#14]
Since SocketRocket also works on a Mac, would it be possible to use the plugin on Cordova/Mac?

**Comment by chax** [comment]
No. Only supported platforms are Android and iOS, but if you find a way to make it work on OS X platform, feel free to submit a PR.
